Revit_oM/Settings/RevitSettings.cs

```diff
@@ -26,31 +26,31 @@
 
 namespace BH.oM.Adapters.Revit.Settings
 {
+    [Description("General settings that are applicable to all actions performed by the instance of Revit adapter.")]
     public class RevitSettings : BHoMObject
     {
         /***************************************************/
         /****             Public Properties             ****/
         /***************************************************/
 
+        [Description("Socket connection settings.")]
```

Comment:
This description does not add anything. If the class description is enough, leave this empty.